certif/tests/integration/components/session-summary-list_test.js

```diff
@@ -147,5 +148,59 @@ module('Integration | Component | session-summary-list', function (hooks) {
       // then
       assert.dom('a[href="/sessions/123"]').exists();
     });
+
+    module('when there is at least one effective candidate', function () {
+      test('it should disable the delete button', async function (assert) {
+        // given
+        this.goToSessionDetailsSpy = sinon.stub();
+        const store = this.owner.lookup('service:store');
+        const sessionSummary = run(() =>
+          store.createRecord('session-summary', {
+            id: 123,
+            enrolledCandidatesCount: 1,
+            effectiveCandidatesCount: 1,
+          })
+        );
+        const sessionSummaries = [sessionSummary];
+        sessionSummaries.meta = {
+          rowCount: 1,
+        };
+        this.sessionSummaries = sessionSummaries;
+
+        // when
+        const screen = await renderScreen(hbs`<SessionSummaryList
+                  @sessionSummaries={{this.sessionSummaries}}
+                  @goToSessionDetails={{this.goToSessionDetailsSpy}} />`);
+
+        // then
+        assert.dom(screen.getByRole('button', { name: 'Supprimer la session 123' })).isDisabled();
+      });
+    });
+
+    module('when there are no effective candidates', function () {
+      test('it should display the delete button', async function (assert) {
+        // given
+        this.goToSessionDetailsSpy = sinon.stub();
+        const store = this.owner.lookup('service:store');
+        const sessionSummary = run(() =>
+          store.createRecord('session-summary', {
+            id: 123,
+          })
+        );
```

Comment:
le run n'est pas necessaire
```suggestion
        const sessionSummary = store.createRecord('session-summary', { id: 123 })
```